src/xokdinst.rs

```diff
@@ -42,6 +42,7 @@ enum InstallConfigPlatform {
 struct InstallConfigMachines {
     name: String,
     replicas: u32,
+    platform: Option<SerdeYamlMap>,
```

Comment:
I think it'd be better to instead to do

```
    #[serde(flatten)]
    extra: SerdeYamlMap,
```

like we do in other cases to "passthrough" everything we don't know.  Right?

Reply:
makes sense.